inst/include/sampler.h

```diff
@@ -161,6 +162,15 @@ namespace smc {
         historyflags GetHistoryFlags(long n) const {return History[n].GetFlags();}
         /// Returns the Effective Sample Size of the specified particle generation.
         double GetESS(long n) const {return History[n].GetESS();}
+        /// Returns a vector of population objects re-ordered to match ancestral lines of the particle system
+        std::vector<population<Space>> GetApop(void) const;

```

Comment:
This is rather trivial, but should this be `GetAPop` or similar for consistency with the other use of A in method names around here?

Reply:
Done, will keep in mind for next time!